Monika After Story/game/zz_reactions.rpy

```diff
@@ -626,9 +641,11 @@ label mas_reaction_quetzal_plush:
     $ store.mas_filereacts.delete_file(gift_ev.category)
     return
 
-init 5 python:
-    if mas_isMonikaBirthday():
-        addReaction("mas_reaction_promisering", "promisering")
+#This one is added later so the init pipeline can define the anni function
+init 11 python:
+    # only available after 6 months or if it's her birthday, may as well add valentine later
+    if mas_anni.pastSixMonths() or mas_isMonikaBirthday():
```

Comment:
honestly, we should include dialogue where monika rejects the ring. for the time being though, we can just ignore it. Can you add a TODO mentionining monika rejecting the ring instead of just simply ignoring it?